src/Pods/Admin/Settings.php

```diff
@@ -165,11 +169,13 @@ private function update_option( array $settings ) {
 	 * @return array The list of Pods settings fields.
 	 */
 	public function get_setting_fields() {
-		$disabled_text = __( 'This setting is disabled because it is forced through the constant/filter elsewhere.', 'pods' );
-		$current_value = __( 'Current value', 'pods' );
+		$did_init = doing_action( 'init' ) || did_action( 'init' );
```

Comment:
```suggestion
		// Only use translation functions after `init` to prevent a WP core notice.
		$did_init = doing_action( 'init' ) || did_action( 'init' );
```